sdk/tables/azure-data-tables/samples/async_samples/sample_insert_delete_entities_async.py

```diff
@@ -65,7 +65,7 @@ async def delete_entity(self):
         from azure.core.credentials import AzureNamedKeyCredential
 
         credential = AzureNamedKeyCredential(self.account_name, self.access_key)
-        table_client = TableClient(endpoint=self.endpoint, credential=credential, table_name=self.table_name)
+        table_client = TableClient(endpoint=self.endpoint, table_name=self.table_name, credential=credential)
```

Comment:
nit: does this have to be the last keyword?

Reply:
It doesn't, but given the `endpoint, table_name` are the two positional parameters - I thought for the samples it would be best to demonstrate the correct order. I didn't bother swapping them in the tests ;)